### ISSUE TITLE
remove width limit

### ISSUE THREAD

**Issue by andreineculau** [tomasbjerre/pull-request-notifier-for-bitbucket#55]
is there a good reason for limiting the width of form.config-area?

filters, URLs (and POST content) can get really long

PS: similarly, the triggers can just be

```
<fieldset>
<label>...</label>
<label>...</label>
...
</fielset>
```

with CSS for label `float: left`

**Comment by tomasbjerre** [comment]
No, the width can be extended :)

**Comment by andreineculau** [comment]
extended, like to 100%? :)

ps: that actually is true for the textarea (POST content), no matter what
you decide about the form.config-area widget

On Thu, Aug 20, 2015 at 4:35 PM, Tomas Bjerre <notifications@github.com>
wrote:

> No, the width can be extended :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tomasbjerre/pull-request-notifier-for-stash/issues/55#issuecomment-133032126>
> .
>



-- 
andreineculau.com <http://www.andreineculau.com>

**Comment by tomasbjerre** [comment]
Released in 2.18, is that ok?

**Comment by andreineculau** [comment]
I cannot confirm, as I have changed jobs to a non-BitBucket env. Thank you anyway, for this and the addon in general!
